### ISSUE TITLE
Top Pages / Search menus (not launch-blocking): Ensure all links are relative to environment

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#11286]
## Description
re: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10982

### Search tools area of page
All  links appear to link to the _production environment_. This needs to be changed.
URLs are entered this way in the Drupal config: https://staging.cms.va.gov/admin/structure/menu/manage/other-search-tools

### Top Pages list
Mostly all links appear to link to _production environment_, due to URLs entered with Prod URLs in the CMS ([example](https://staging.cms.va.gov/admin/structure/menu/item/11455/edit?destination=/admin/structure/menu/manage/popular-on-va-gov)) 

These four exceptions are not currently understood:

- File For disability compensation
- Schedule or manage health appointments
- Refill or track a prescription. 
- Compare GI Bill benefits

... all of which are relative to their environment, despite va.gov in the menu item config. All other links to be changed to use relative URLs. 
* Update menu config to use relative links
* Figure out why the 4 links mentioned _don't_ point to prod, despite menu item URLs pointing to prod

### Engineering notes

- We already checked the graphQL debugger/output, and it looks like it's working as expected.

## Caveat
If some of these URLs point to pages outside the CMS / modernized env, they will need to stay absolute. Otherwise, relative links will 404 on lower environments. 

## Acceptance criteria
- [ ] Homepage menu links are relative to the environment
- [ ] Absolute links which are content in Drupal have been fixed (@jilladams)
- [ ] Four non-content links in "Top pages" list (mysterious) are corrected

**Comment by jilladams** [comment]
Confirmed with Michelle: https://dsva.slack.com/archives/C03NSH4SLQY/p1667852499663229

Staging links should keep the user in Staging; Prod links in Prod. And selecting a node as link in the CMS will manage this for us. So: this will probably close as no-op, but let's keep it to confirm when Drupal content is created.

**Comment by jilladams** [comment]
Adding to sprint and asked Bryan to confirm that in Staging, all links are relative to staging post-drupal. If so: we can close.

**Comment by apisandipas** [comment]
## Audit of Homepage Links on New Homepage

This was an audit perform to assure that all links in the new homepage are made relative to their environment.

### Hero section ✅
All links are relative.

### Search ❌
All  links appear to link to the _production environment_. This need to be changed.

### Top Pages ❌
Mostly all links appear to link to _production environment_, with the exception of:

- File For disability compensation
- Schedule or manage health appointments
- Refill or track a prescription. 
- Compare GI Bill benefits

... all of which are relative to thier environment. All other links to be change to use relative urls. 

### News Spotlight ✅

These links are external and appear as they should. 

### Benefits Hubs ✅
 These links are all relative to their environment.

**Comment by jilladams** [comment]
Thank you! Updated your comment with some add'l notes

**Comment by jilladams** [comment]
Also updated ticket body with the changes that need to be made to fix this up. @wesrowe should be added to #11942

**Comment by jilladams** [comment]
Consideration: in the menu, should we focus on the editor experience for how we manage relative vs. absolute links in menus. (Relevant to this issue, but applies to more menus -- header, footer, etc.)

**Comment by jilladams** [comment]
https://dsva.slack.com/archives/C52CL1PKQ/p1671486511259009
Bryan's hunch: including www. in the URL for menu items will allow the link to behave correctly relative to the environment. 
I did update these menu links -- the 4 listed that behave strangely all _did_ include `www.` already, and the rest didn't. I've added `www.` where it was missing. 

The implication is: once those content updates get pushed out (not sure what it takes for them to hit staging, have asked [here](https://dsva.slack.com/archives/CDHBKAL9W/p1671487333298709)), this ticket may be complete & not need any further Drupal work. FYI @chri5tia
